### PR TITLE
i18n(es): Translated `from-gatsby.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translate `from-gatsby.mdx` to spanish

### CODE REVIEW

File: src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx [hunk 1]
```diff
@@ -0,0 +1,578 @@
...
+
+2. **Renombra** la carpeta `static/` de Gatsby a `public/`, y úsala como la carpeta `public/` de Astro. 
+
+    Astro utiliza una carpeta llamada `public/` para los activos estáticos. Como alternativa, puedes copiar el contenido de `static/` ien la carpeta `public/` existente de Astro.
+
+3. **Copia o mueve** los otros archivos y carpetas de Gatsby (p. ej. `components`, `pages`, etc.)  según sea necesario en tu carpeta `src/` de Astro a medida que reconstruyes tu sitio, siguiendo la [estructura de proyecto de Astro](/es/core-concepts/project-structure/).
+
+  La carpeta `src/pages/` de Astro es una carpeta especial utilizada para la creación de páginas y publicaciones de tu sitio mediante enrutamiento basado en archivos a partir de archivos `.astro`, `.md` y `.mdx`. No será necesario configurar ningún comportamiento de enrutamiento para tus archivos Astro, Markdown y MDX en Astro.
+
+  Todas las demás carpetas son opcionales y puedes organizar el contenido de tu carpeta `src/` de la forma que desees. Otras carpetas comunes en proyectos de Astro incluyen `src/layouts/`, `src/components`, `src/styles`, y `src/scripts`.
```
Comment (dreyfus92) [inline src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx:111]:
```suggestion
Todas las demás carpetas son opcionales y puedes organizar el contenido de tu carpeta `src/` de la forma que desees. Otras carpetas comunes en proyectos de Astro incluyen `src/layouts/`, `src/components`, `src/styles`, y `src/scripts`.
```

File: src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx [hunk 2]
```diff
@@ -0,0 +1,578 @@
...
+
+### Consejos: Convierte archivos JSX a archivos `.astro`
+
+Aquí tienes algunos consejos para convertir un componente de Gatsby con extensión `.js` a un componente  `.astro`:
```
Comment (dreyfus92) [inline src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx:115]:
```suggestion
Aquí tienes algunos consejos para convertir un componente de Gatsby con extensión `.js` a un componente `.astro`:
```

File: src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx [hunk 3]
```diff
@@ -0,0 +1,578 @@
...
+
+En componentes de React (`.jsx`), utiliza la sintaxis estándar de JSX para imágenes (`<img />`). Astro no optimizará estas imágenes, pero puedes instalar y utilizar paquetes de NPM para obtener más flexibilidad.
+
+Puedes obtener más información sobre [cómo usar imágenes en Astro](/es/guides/images/) en la Guía de Imágenes.
+
+### Gatsby GraphQL a Astro
+
+Eliminar todas las referencias a consultas de GraphQL y, en su lugar, utilizar [`Astro.glob()`](/es/guides/imports/#astroglob) para acceder a los datos de tus archivos locales.
+
+O, si estás utilizando colecciones de contenido, consulta tus archivos Markdown y MDX en `src/content/` utilizando  [`getEntry()` y `getCollection()`](/es/guides/content-collections/#querying-collections).
```
Comment (dreyfus92) [inline src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx:358]:
```suggestion
O, si estás utilizando colecciones de contenido, consulta tus archivos Markdown y MDX en `src/content/` utilizando [`getEntry()` y `getCollection()`](/es/guides/content-collections/#querying-collections). 
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo @Waxer59, te dejo algunas sugerencias 😁

**Comment by Waxer59** [comment]
Gracias por la revisión 🤠

File: src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx [hunk 4]
```diff
@@ -0,0 +1,578 @@
...
+---
+import { Image } from '@astrojs/image/components';
+import localImage from "../images/brand/logo.png";
+import rocket from '../images/rocket.svg';
+---
+<Image src={localImage} alt="The Astro Logo" />
+<img src={rocket} alt="A rocketship in space.">
+```
+
+Los componentes `<Image />` y `<Picture />` de Astro son experimentales y actualmente solo funcionan en archivos `.astro` y `.mdx` files only. Puedes ver una [lista completa de atributos de componente](/es/guides/integrations-guide/image/#usage) disponibles para estos componentes, y debes tener en cuenta que varios atributos difieren de los atributos de Gatsby.
```
Comment (Yan-Thomas) [inline src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx:336]:
```suggestion
Los componentes `<Image />` y `<Picture />` de Astro son experimentales y actualmente solo funcionan en archivos `.astro` y `.mdx` files only. Puedes ver una [lista completa de atributos de componente](/es/guides/integrations-guide/image/#uso) disponibles para estos componentes, y debes tener en cuenta que varios atributos difieren de los atributos de Gatsby.
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the great work 💜

**Pull request merged**